<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoFechas</class>
 <widget class="QWidget" name="DialogoFechas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogoFechas</string>
  </property>
  <widget class="QDialogButtonBox" name="btnbox_fechas">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_general">
    <item>
     <layout class="QFormLayout" name="layout_opciones">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_etiqueta">
        <property name="text">
         <string>etiqueta</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineedit_etiqueta"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_fecha">
        <property name="text">
         <string>fecha</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateedit_fecha">
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::DaySection</enum>
        </property>
        <property name="displayFormat">
         <string>d/M/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_agregar_fecha">
        <property name="text">
         <string>agregar</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_limpiar_fecha">
        <property name="text">
         <string>limpiar</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btn_eliminar_fecha">
        <property name="text">
         <string>eliminar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layout_lista">
      <item>
       <widget class="QListWidget" name="lista_fechas"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="action_guardar_fecha">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>guardar_fecha</string>
   </property>
  </action>
  <action name="action_limpiar_fecha">
   <property name="text">
    <string>limpiar_fecha</string>
   </property>
  </action>
  <action name="action_eliminar_fecha">
   <property name="text">
    <string>eliminar_fecha</string>
   </property>
  </action>
  <action name="action_actualizar_y_cerrar">
   <property name="text">
    <string>actualizar_y_cerrar</string>
   </property>
  </action>
  <action name="action_estado_btn_eliminar">
   <property name="text">
    <string>estado_btn_eliminar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lista_fechas</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>action_estado_btn_eliminar</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnbox_fechas</sender>
   <signal>accepted()</signal>
   <receiver>action_actualizar_y_cerrar</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnbox_fechas</sender>
   <signal>rejected()</signal>
   <receiver>DialogoFechas</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_agregar_fecha</sender>
   <signal>released()</signal>
   <receiver>action_guardar_fecha</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_eliminar_fecha</sender>
   <signal>released()</signal>
   <receiver>action_eliminar_fecha</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_limpiar_fecha</sender>
   <signal>released()</signal>
   <receiver>action_limpiar_fecha</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
